packages/frontity-org-theme/src/processors/mobile-desktop.ts

```diff
@@ -0,0 +1,40 @@
+import { css } from "frontity";
+
+const createProcessors = (options: {
+  [className: string]: ReturnType<typeof css>;
+}) => {
+  return Object.entries(options).map(([className, style]) => ({
+    name: className,
+    test: ({ node }) =>
+      node.type === "element" &&
+      node.props.className &&
+      node.props.className.split(" ").includes(className),
+    processor: ({ node }) => ({
+      ...node,
+      props: {
+        ...node.props,
+        css: css`
+          ${node.props.css}
+          ${style}
+        `
+      }
+    })
+  }));
+};
+
+export default createProcessors({
+  "mobile-only": css`
+    display: none;
+
+    @media only screen and (max-width: 768px) {
+      display: block;
```

Comment:
Not all elements will have `display: block`, some will have `inline` or `inline-block`, but I have absolutely no idea how to do this. 

Maybe `display: inherit` works for this case?  I'm going to move this to `blocked` until we figure it out.

cc: @DAreRodz @michalczaplinski

Reply:
The same can be written just to hide the element, reversing the media queries:

```ts
export default createProcessors({
  "mobile-only": css`
    @media only screen and (min-width: 769px) {
      display: none;
    }
  `,
  "desktop-only": css`
    @media only screen and (max-width: 768px) {
      display: none;
    }
  `
});

Reply:
yeah, I think `display: inherit` should work here. We would just have to kind of "do the opposite" in CSS like:

```css
"mobile-only": css`
    display: inherit; 

    /* if we are on a BIG (>769px) screen hide it */
    @media only screen and (min-width: 769px) {
      display: none;
     }
`,
"desktop-only": css`
    display: inherit;
    
    /* if we are on a SMALL (<768px) screen hide it */
    @media only screen and (max-width: 768px) {
      display: none;
    }
  `
```

Reply:
Thanks guys!

I was wrong with `inherit`. It [gets the property from the parent](https://www.w3schools.com/cssref/css_inherit.asp). So in a case like this:
```html
<div><span>Span 1 </span><span>Span 2</span></div>
<div><span style="display: inherit;">Span 3 </span><span>Span 4</span></div>
```
the third `span` will get a `display: block` instead of a `display: inline` and the result is:
```
Span 1 Span 2
Span 3
Span 4
```
[Codesandox](https://codesandbox.io/s/display-inherit-6t67p)

@iamuchejude let's try with the opposite query, as @DAreRodz and @michalczaplinski suggested 👍 

Unblocking this now :)

Reply:
oh, I didnt see your comment before I posted mine @DAreRodz haha